src/angulartics-webtrends-analytics.js

```diff
@@ -0,0 +1,118 @@
+(function (window, angular) {
+    'use strict';
+
+    /**
+     * @ngdoc overview
+     * @name angulartics.webtrends.analytics
+     * Enables analytics support for Webtrends Analytics (http://webtrends.com/products-solutions/analytics/)
+     */
+    angular.module('angulartics.webtrends.analytics', ['angulartics'])
+        .config(['$analyticsProvider', function ($analyticsProvider) {
+
+            // Depending on Tag version, the vendor name will be different.
```

Comment:
Is this possibly more configurable for the user so they don't need to modify this src? Other than that this is a great addition thanks!